cheatsheets/Content_Security_Policy_Cheat_Sheet.md

```diff
@@ -82,27 +88,40 @@ Navigation directives instruct the browser about the locations that the document
 
 ## Reporting Directives
 
-Reporting directives deliver violation of prevented behaviors to specified locations. These directives serve no purpose on their own and are dependent on other directives.
+Reporting directives deliver violations of prevented behaviors to specified locations. These directives serve no purpose on their own and are dependent on other directives.
 
 - `report-to` which is a groupname defined in the header in a json formatted header value. Does not have proper browser support yet.
-  - [MDN report-to documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-to#Examples)
-- `report-uri` directive will be getting deprected by `report-to`, which is a URI that the reports are sent to.
+  - [MDN report-to documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-to)
+- `report-uri` directive is deprecated by `report-to`, which is a URI that the reports are sent to.
   - Goes by the format of: `Content-Security-Policy: report-uri https://example.com/csp-reports`
 
 In order to ensure backward compatibility, use the 2 directives in conjonction. Whenever a browser supports `report-to`, it will ignore `report-uri`. Otherwise, `report-uri` will be used.
 
 ## Special Directive Sources
 
-| Value           | Description                                                     |
-|-----------------|-----------------------------------------------------------------|
-| 'none'          | No URLs match.                                                  |
-| 'self'          | Refers to the origin site with the same scheme and port number. |
-| 'unsafe-inline' | Allows the usage of inline scripts or styles.                   |
-| 'unsafe-eval'   | Allows the usage of eval in scripts.                            |
+| Value            | Description                                                                                                           |
+|------------------|-----------------------------------------------------------------------------------------------------------------------|
+| 'none'           | No URLs match.                                                                                                        |
+| 'self'           | Refers to the origin site with the same scheme and port number.                                                       |
+| 'unsafe-inline'  | Allows the usage of inline scripts or styles.                                                                         |
+| 'unsafe-eval'    | Allows the usage of eval in scripts.                                                                                  |
+| 'strict-dynamic' | Informs the browser to trust scripts originating from a root trusted script.                                        |
+
+*Note:* `strict-dynamic` is not a standalone directive and should be used in combination with other directive values, such as `https:`, `nonce`, `hashes`, etc.
```

Comment:
Note that 'https:' and any other host whitelist is ignored in the presence of 'strict-dynamic'.
This is to ensure backward compatibility with CSP2 browsers and older browsers.
See:
 - https://speakerdeck.com/lweichselbaum/csp-a-successful-mess-between-hardening-and-mitigation?slide=51
 - https://speakerdeck.com/lweichselbaum/csp-a-successful-mess-between-hardening-and-mitigation?slide=52

Reply:
I am considering advising them to have 2 separate policies, one for older versions, and the other for the latest. That'll set both in action, or depending on what their browsers support. What do you think?

Reply:
I like this idea.

Reply:
which two policies would you advise? One with and one without fallbacks?
the nice thing about having a single policy with fallbacks is that you don't need to do user agent sniffing which can become very tricky and if you for example opt out safari now you must become active once they start supporting new things while if you have the fallbacks in the policy every browser that supports or adds support for the newer keywords will just apply them and ignore the fallbacks.

Reply:
Sounds reasonable. I'll be adding an extra policy in the examples.